packages/playground/assets/css/playground.css

```diff
@@ -19,13 +19,12 @@
 }
 
 #wp-playground-toolbar {
-	background-color: #eaaa00;
-	font-weight: bold;
-	text-align: center;
-	font-size: 1rem;
-	padding: 0.75em;
+	background-color: #fff7cc;
+	font-family: "Andale mono", Helvetica, Monospace, Arial;
```

Comment:
Why this font? I would prefer if we stick to the same font like WordPress.

Reply:
Also, the font creates some alignment issues that don't exist with the default WordPress font.

![image](https://github.com/WordPress/playground-tools/assets/1199991/2863782d-7eeb-4b6b-b875-85d57a2f75b0)

Reply:
> Why this font? I would prefer if we stick to the same font like WordPress.

It's the same font list used in the existing notice: https://github.com/WordPress/wordpress-playground/blob/02a528dd04996d7c9c2a1fdae143ee39c60933f0/packages/playground/website/src/components/browser-chrome/style.module.css#L13.

My thinking was to match Playground's existing notice. Didn't think about overall consistency with the WordPress Admin, but it's a good point.
My question would be, do you want it to look as part of the Admin or as part of Playground?

> alignment issues

I'll sort those once there's a decision on the font. Okay?

Reply:
> I'll sort those once there's a decision on the font. Okay?

Sure, I think that using the WP font is the best solution for now.